spec/support/test_env.rb

```diff
@@ -81,7 +80,7 @@ def setup_factory_repo
     end
 
     # We must copy bare repositories because we will push to them.
-    system(*%W(git clone --bare #{factory_repo_path} #{factory_repo_path_bare}))
+    system(*%W(git clone -q --bare #{factory_repo_path} #{factory_repo_path_bare}))
```

Comment:
Would there be a way to make this line under 83 characters? (Im not sure there is, so if none, we will ignore this one)